zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -140,7 +140,7 @@ zipkin:
     # Default is "match all websites"
     instrumented: .*
     # URL placed into the <base> tag in the HTML
-    base-path: /zipkin/
+    base-path: /zipkin
```

Comment:
ps people will set this with properties.. if there's a behavior change we might want to attack that in the UiConfiguration class